tests/cpp_integration_tests/test_main_model_sc.cpp

```diff
@@ -17,7 +17,8 @@ using enum CalculationSymmetry;
 TEST_CASE("Test main model - short circuit") {
     MainModel main_model{50.0, meta_data::meta_data_gen::meta_data};
 
-    SUBCASE("Single node + source") {
+    SUBCASE("Single node + source") { // TODO(mgovers): existing validation case for this is too difficult. we may want
+                                      // to keep this. Also should not live here but in SC solver
```

Comment:
can you elaborate why it cannot be in the validation case

Reply:
these TODOs were all added before our discussion last week Thursday and were a proposal. I have not updated the comments due to time constraints and priority. Note that this PR is still in draft for that very same reason.

To elaborate why my initial proposal was the way it was:

1. the current test code shows how the values were obtained. moving them will make those magic values that only the people with an engineering background will be able to understand. it's alright to have some of them, but it will become impossible to debug for someone who doesn't have that knowledge.
2. IMO, the main model is not the right location for these tests, but nor are the API tests. See my other comment.

Reply:
cfr. agreement: scoped to follow-up PR